File: APIQuanLyCuaHang/APIQuanLyCuaHang/ViewModels/DataChart/StaffDC.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace APIQuanLyCuaHang.ViewModels.DataChart
{
    public class StaffDC
    {
        public int MaNv { get; set; }

        public string HoTen { get; set; } = null!;

        public string GioiTinh { get; set; } = null!;

        public DateOnly? NgaySinh { get; set; }

        public string? DiaChi { get; set; }

        public string? Cccd { get; set; }

        public string Sdt { get; set; } = null!;

        public string Email { get; set; } = null!;

        public DateOnly NgayVaoLam { get; set; }

        public string? TenTaiKhoan { get; set; }

        public string MatKhau { get; set; } = null!;

        public string? TinhTrang { get; set; }

        public bool? IsDelete { get; set; }

        public int? MaChucVu { get; set; }
        public decimal? DoanhThuMangLai { get; set; }
    }
}

File: QuanLyCuaHangUXUI/QuanLyCuaHangProject/src/models/dto/dataCharts/StaffDC.js
export default class StaffDC {
  constructor(
    maNv,
    hoTen,
    gioiTinh,
    ngaySinh = null,
    diaChi = null,
    cccd = null,
    sdt,
    email,
    ngayVaoLam,
    tenTaiKhoan = null,
    matKhau,
    tinhTrang = null,
    isDelete = null,
    maChucVu = null,
    soDonHangDamNhan = null,
    doanhThuMangLai = null,
  ) {
    this.maNv = maNv
    this.hoTen = hoTen
    this.gioiTinh = gioiTinh
    this.ngaySinh = ngaySinh // Ngày sinh có thể là null
    this.diaChi = diaChi // Địa chỉ có thể là null
    this.cccd = cccd // CCCD có thể là null
    this.sdt = sdt
    this.email = email
    this.ngayVaoLam = ngayVaoLam // Ngày vào làm không thể là null
    this.tenTaiKhoan = tenTaiKhoan // Tên tài khoản có thể là null
    this.matKhau = matKhau
    this.tinhTrang = tinhTrang // Tình trạng có thể là null
    this.isDelete = isDelete // IsDelete có thể là null
    this.maChucVu = maChucVu // Mã chức vụ có thể là null
    this.soDonHangDamNhan = soDonHangDamNhan // Số đơn hàng đã nhận có thể là null
    this.doanhThuMangLai = doanhThuMangLai // Doanh thu mang lại có thể là null
  }
}
